### ISSUE TITLE
Deleting an aiven_account_team_member does not work

### ISSUE THREAD

**Issue by benoit-parmentier-2** [aiven/terraform-provider-aiven#640]
Hello,

We use Aiven terraform provider to manage our users and teams.

```
terraform {
  required_version = ">= 1.0.3"
  experiments      = [module_variable_optional_attrs]
  required_providers {
    aiven = {
      source  = "aiven/aiven"
      version = "2.4.0"
    }
  }
}
```

When we try to remove an user from a team, terraform indicates that the user is removed from the team (Destroying...)
But, the user remains in Aiven console.

```
1778  - resource "aiven_account_team_member" "avn_team_member" {
1779      - accepted              = false -> null
1780      - account_id            = "xxxxx" -> null
1781      - create_time           = "2021-06-08 20:08:03 +0000 UTC" -> null
1782      - id                    = "xxx/yyy/xxx@yyy.com" -> null
1783      - invited_by_user_email = "[MASKED]" -> null
1784      - team_id               = "xxxxxx" -> null
1785      - user_email            = "xxx@yyy.com" -> null
1786    }
```

The documentation (https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/account_team_member) explains that `it will also eliminate an account team member if one has accepted an invitation previously.`, but it seems that it does not work.

Sincerely,
Benoît

**Comment by ivan-savciuc** [comment]
@benoit-parmentier-2 where exactly are you looking to see a change in the Aiven console? If you are checking the `https://console.aiven.io/accounts/XXX/teams` URL, it shows a list of account teams, not team members.

**Comment by benoit-parmentier-2** [comment]
Fixed in version 2.7.1
